test/unit/DCAStrategies/dca-strategies-positions-handler.spec.ts

```diff
@@ -82,6 +86,29 @@ contract('DCAStrategiesPositionsHandler', () => {
       then('transferFrom() is called correctly', async () => {
         expect(tokenA.transferFrom).to.have.been.calledOnceWith(user.address, DCAStrategiesPositionsHandlerMock.address, toDeposit);
       });
+      then('_approveHub() is called correctly', async () => {
```

Comment:
`_approveHub` is quite complex. I think that we should test is separately, and make sure that it was called correctly from `deposit`. Because we will need to do the same in `increase`